scripts/analyze_telemetry.sh

```diff
@@ -18,7 +18,7 @@ top() {
 
 get_func() { jq -r "select(.name == \"$1\")"; }
 
-mean() { jq -sr '[.[].duration] | add/length'; }
+mean() { jq -sr '[.[].duration] | add/length | select(.duration != null and .duration != 0)'; }
```

Comment:
```suggestion
mean() { jq -sr '[.[].duration] | select(length > 0) | add/length'; }
```
Check `length` before the division
https://stackoverflow.com/questions/26195214/how-to-nicely-remove-empty-array-in-jq